management/persistence.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Persistence
+nav_order: 1
+description: How FalkorDB writes data to disk
+parent: Management
+---
+
+Persistence refers to the writing of data to durable storage, such as a solid-state disk (SSD). FalkorDB provides a range of persistence options. These include:
+
+* **RDB** (FalkorDB Database): RDB persistence performs point-in-time snapshots of your dataset at specified intervals.
+* **AOF** (Append Only File): AOF persistence logs every write operation received by the server. These operations can then be replayed again at server startup, reconstructing the original dataset. Commands are logged using the same format as the FalkorDB protocol itself.
+* **No persistence**: You can disable persistence completely. This is sometimes used when caching.
```

Comment:
I think this gos without saying.
Someone who's reading this paragraph is most likely interested in persistency.